config.go

```diff
@@ -7,9 +7,35 @@ import (
 	"time"
 )
 
+// These are the versions of the protocol (which includes wire protocol as
```

Comment:
I'll clarify this comment it's just Raft-specific log entries and RPC messages. It doesn't currently apply to snapshots, though we could eventually use it there if we needed to.